src/c++/perf_analyzer/genai-perf/tests/test_llm_metrics.py

```diff
@@ -476,10 +478,10 @@ def test_llm_metrics_get_base_name(self) -> None:
                         "request_inputs": {"text_input": "This is test"},
                         "response_timestamps": [7, 8, 13, 18],
                         "response_outputs": [
-                            {"text_output": "cats"},
-                            {"text_output": "are"},
-                            {"text_output": "cool"},
-                            {"text_output": "too"},
+                            {"text_output": "cat"},
+                            {"text_output": " is"},
+                            {"text_output": " cool"},
```

Comment:
why is the extra space necessary here?

Reply:
It's just to match what models return in streaming case. A lot of times, they have whitespace in front since it's sliced per token.